@eugene123tw could you have a look at iseg tests failure?
```
    results_per_confidence = self._get_results_per_confidence(
  File "/home/validation/actions-runner/_work/training_extensions/training_extensions/.tox/integration-test-instance_segmentation/lib/python3.10/site-packages/otx/core/metrics/fmeasure.py", line 233, in _get_results_per_confidence
    result_point = self.evaluate_classes(
  File "/home/validation/actions-runner/_work/training_extensions/training_extensions/.tox/integration-test-instance_segmentation/lib/python3.10/site-packages/otx/core/metrics/fmeasure.py", line 279, in evaluate_classes
    metrics, counters = self.get_f_measure_for_class(
  File "/home/validation/actions-runner/_work/training_extensions/training_extensions/.tox/integration-test-instance_segmentation/lib/python3.10/site-packages/otx/core/metrics/fmeasure.py", line 323, in get_f_measure_for_class
    batch_pred_bboxes = self.__filter_pred(
  File "/home/validation/actions-runner/_work/training_extensions/training_extensions/.tox/integration-test-instance_segmentation/lib/python3.10/site-packages/otx/core/metrics/fmeasure.py", line 381, in __filter_pred
    keep = (entity.labels == label_idx) & (entity.score > confidence_threshold)
RuntimeError: The size of tensor a (66) must match the size of tensor b (100) at non-singleton dimension 0
Traceback (most recent call last):
```

--- a/src/otx/algo/instance_segmentation/segmentors/maskrcnn_tv.py
+++ b/src/otx/algo/instance_segmentation/segmentors/maskrcnn_tv.py
@@ -91,6 +91,7 @@ class MaskRCNN(_MaskRCNN):
         for i, (pred, scale_factor, ori_shape) in enumerate(zip(result, scale_factors, ori_shapes)):
             boxes = pred["boxes"]
             labels = pred["labels"]
+            scores = pred["scores"]
             _scale_factor = [1 / s for s in scale_factor]  # (H, W)
             boxes = boxes * boxes.new_tensor(_scale_factor[::-1]).repeat((1, int(boxes.size(-1) / 2)))
             h, w = ori_shape
@@ -99,8 +100,10 @@ class MaskRCNN(_MaskRCNN):
             keep_indices = (boxes[:, 2] - boxes[:, 0]) * (boxes[:, 3] - boxes[:, 1]) > 0
             boxes = boxes[keep_indices > 0]
             labels = labels[keep_indices > 0]
+            scores = scores[keep_indices > 0]
             result[i]["boxes"] = boxes
             result[i]["labels"] = labels - 1  # Convert back to 0-indexed labels
+            result[i]["scores"] = scores
             if "masks" in pred:
                 masks = pred["masks"][keep_indices]
                 masks = paste_masks_in_image(masks, boxes, ori_shape)

